site/src/analytics-os-boards.ts

```diff
@@ -12,6 +12,7 @@ const friendlyBoardName: Record<string, string> = {
   "odroid-xu4": "ODROID XU4",
```

Comment:
Should we name "OVA" "Virtual Machine" ?

Reply:
Yes <https://github.com/home-assistant/analytics.home-assistant.io/pull/169>